lms/static/js/student_account/views/FormView.js

```diff
@@ -172,7 +172,7 @@
 
                         if (key) {
                             test = this.validate(elements[i]);
-                            if (test.isValid) {
+                            if (test.isValid || $el.attr('id') === 'login-email') {
```

Comment:
In the code there are two types of login. The ```login.html``` and the ```login_and_register.html```. Probably the ```login.html``` is gonna disappear at some point.